security-oauth2/src/main/java/io/micronaut/security/oauth2/configuration/OauthClientConfigurationProperties.java

```diff
@@ -25,29 +25,17 @@
 import io.micronaut.http.MediaType;
 import io.micronaut.security.oauth2.client.clientcredentials.ClientCredentialsConfiguration;
 import io.micronaut.security.oauth2.client.clientcredentials.propagation.ClientCredentialsHeaderTokenPropagatorConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.AuthorizationEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.DefaultEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.DefaultSecureEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.EndSessionEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.EndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.IntrospectionEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.RevocationEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.SecureEndpointConfiguration;
-import io.micronaut.security.oauth2.configuration.endpoints.TokenEndpointConfiguration;
+import io.micronaut.security.oauth2.configuration.endpoints.*;
 import io.micronaut.security.oauth2.endpoint.authorization.request.Display;
 import io.micronaut.security.oauth2.endpoint.authorization.request.OpenIdScope;
 import io.micronaut.security.oauth2.endpoint.authorization.request.Prompt;
 import io.micronaut.security.oauth2.endpoint.authorization.request.ResponseType;
 import io.micronaut.security.oauth2.grants.GrantType;
+import io.micronaut.security.token.propagation.AbstractOutgoingRequestProcessorMatcher;
+
 import java.net.URL;
 import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.regex.Pattern;
+import java.util.*;
```

Comment:
are the use of these wildcard imports intentional?

Reply:
It was the IDE optimising them. Wildcard imports are allowed as per our Checkstyle config.